### ISSUE TITLE
Question about train/validation split

### ISSUE THREAD

**Issue by anands-repo** [google/deepvariant#224]
Hi,

I had a question regarding how train/validation split was determined for training the models.

From the DeepVariant paper, I noticed that chromosome 1-19 was used for training. Within chr1-19, is there a reason to prefer one of the following training-validation splits:
1. Designate chr-a, 1 <= a <= 19, as validation set, and remaining as training set.
2. Use some random subset of all labeled images as a validation set, and the disjoint set as training set (the random split may be grouped by sites, or not).

Specifically, I am interested in knowing whether there was a reason for preferring one split over the other, or whether it is an arbitrary choice, that is considered equivalent.

Thanks!

**Comment by AndrewCarroll** [comment]
Hi @anands-repo 

The major factor was the length of chromosomes, selecting sizes that would allow enough data for comparison of tune and test while still maximizing the data available for training. The tuning and test chromosomes should also be roughly representative of the overall genome content.

There are some chromosomes that should probably be avoided for tune or test choices - specifically ones that have regions quite different from the rest of the genome. Chromosome 6 is an example, since it contains the MHC regions.

So it is mostly arbitrary, though other methods such as Clairvoyante seem to have adopted the convention as well. It is nice for the choices to align, since it allows comparisons with the same regions.
